app/api/api_v1/schemas/search.py

```diff
@@ -165,6 +165,8 @@ class SearchResponseFamily(BaseModel):
     family_slug: str
     family_name: str
     family_description: str
+    family_category: str
```

Comment:
Can't see any tests updated to check we have these new fields in the response - perhaps something for the next PR?

Reply:
yeah, good call, we should add this to the "big list o' things to test!"